### DISCUSSION TITLE
Geomap dynamic Symbol

### DISCUSSION THREAD

**Discussion by yosiasz** [grafana/grafana#56886] [Plugins]
Greetings

Would be real awesome if the Symbol in geomap could be set dynamically based on data

![image](https://user-images.githubusercontent.com/900951/195641346-994de200-5c8c-4b1c-89d2-5ce62f02b8df.png)

For example, if one manages different kind of devices and we want to plot this on a map it would be nice to be able to see clusters of specific device types on a map for a quick visual check/analysis based on the symbol without having to hover over each map point.

So if you have tel towers of a specific kind clustered in the NW, but you dont have enough of those in the SW you can get an idea with a quick glance of the symbols.

Any future plans for this feature?

Thanks

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
